apps/backend/src/api/handlers/createBuild.ts

```diff
@@ -257,7 +257,7 @@ async function createBuildFromRequest(ctx: BuildContext) {
     prNumber: body.prNumber ?? null,
     prHeadCommit: body.prHeadCommit ?? null,
     referenceCommit: body.referenceCommit ?? null,
-    referenceBranch: body.referenceBranch ?? null,
+    baseBranch: body.referenceBranch ?? null,
```

Comment:
IMO, it should be named `baselineBranch` but all the codebase would be impacted

Reply:
It's from the API we can't rename it. I renamed it internally to `baseBranch`, it's actually the good name because it's Git related: base / head.